frontend/src/components/SeedlotRegistrationSteps/CollectionStep/index.tsx

```diff
@@ -0,0 +1,320 @@
+          <Column sm={12} md={4} lg={6}>
+            <DatePicker
+              datePickerType="single"
+              dateFormat={DATE_FORMAT}
+              value={state.startDate}
+              onChange={(_e: Array<Date>, selectedDate: string) => {
+                handleFormInput('startDate', selectedDate);
+              }}
+            >
+              <DatePickerInput
+                name={fieldsConfig.startDate.name}
+                placeholder={fieldsConfig.startDate.placeholder}
+                labelText={fieldsConfig.startDate.labelText}
+                helperText={fieldsConfig.startDate.helperText}
+                invalid={validationObj.isStartDateInvalid}
+                invalidText={fieldsConfig.startDate.invalidText}
+                size="md"
+              />
+            </DatePicker>
+          </Column>
+          <Column sm={12} md={4} lg={6}>
+            <DatePicker
+              datePickerType="single"
+              dateFormat={DATE_FORMAT}
+              minDate={state.startDate}
+              value={state.endDate}
+              onChange={(_e: Array<Date>, selectedDate: string) => {
+                handleFormInput('endDate', selectedDate);
+              }}
+            >
+              <DatePickerInput
+                name={fieldsConfig.endDate.name}
+                placeholder={fieldsConfig.endDate.placeholder}
+                labelText={fieldsConfig.endDate.labelText}
+                helperText={fieldsConfig.endDate.helperText}
+                invalid={validationObj.isEndDateInvalid}
+                invalidText={fieldsConfig.endDate.invalidText}
+                size="md"
+              />
+            </DatePicker>
+          </Column>
+        </Row>
+        <Row>
+          <Column sm={12} md={4} lg={6}>
+            <NumberInput
+              name={fieldsConfig.numberOfContainers.name}
+              ref={numberOfContainersInputRef}
+              value={state.numberOfContainers}
+              label={fieldsConfig.numberOfContainers.labelText}
+              invalidText={fieldsConfig.numberOfContainers.invalidText}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                handleFormInput('numberOfContainers', e.target.value);
+              }}
+              hideSteppers
+              disableWheel
+            />
+          </Column>
+          <Column sm={12} md={4} lg={6}>
+            <NumberInput
+              name={fieldsConfig.volumePerContainers.name}
+              ref={volumePerContainersInputRef}
+              value={state.volumePerContainers}
+              label={fieldsConfig.volumePerContainers.labelText}
+              invalidText={fieldsConfig.volumePerContainers.invalidText}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                handleFormInput('volumePerContainers', e.target.value);
+              }}
+              hideSteppers
+              disableWheel
+            />
```

Comment:
On figma, the default value of this field seems to be 1, not sure if that's correct, maybe align with the UX team?
![image](https://user-images.githubusercontent.com/105936322/231475405-fb2ed67e-bc64-481c-8b84-4cffb59bd7b5.png)

Reply:
Fixed. Please review it again.